projects/storefrontlib/src/lib/cms/services/cms-mapping.service.ts

```diff
@@ -29,6 +29,16 @@ export class CmsMappingService {
     return routes;
   }
 
+  getGuardsForComponents(componentTypes: string[]): any[] {
+    const guards = new Set<CanActivate>();
```

Comment:
Why do we have `Set<CanActivate>` here while in other places we use `any[]`?

Reply:
👍